client/components/UserList/components/ListItem/ListItem.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const OnlineStatus = ({ status }) => (
+  <span className={`userlist-status ${status ? 'online' : 'offline'}`}>{status ? 'Online' : 'Offline'}</span>
```

Comment:
I;m not sure you need own component for these one line containers

Reply:
Why not?
I think, in this case, this is better for reading. Also, we can create own component from this and used in another place.